### PR TITLE
httpie 0.9.3

### PR DESCRIPTION
**Pull request by jkbrzt**
https://github.com/jkbrzt/httpie/releases/tag/0.9.3

### CODE REVIEW

File: Library/Formula/httpie.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 class Httpie < Formula
   desc "User-friendly cURL replacement (command-line HTTP client)"
-  homepage "https://github.com/jakubroztocil/httpie"
-  url "https://github.com/jakubroztocil/httpie/archive/0.9.2.tar.gz"
-  sha256 "1f0b0b6563de25bad0ed190d62f1130573124201382824756d015b1422a234cc"
+  homepage "http://httpie.org/"
+  url "https://github.com/jkbrzt/httpie/archive/0.9.3.tar.gz"
+  sha256 "2a0c7cf6a6914620eebc2d7700e8e7a57aabde62bd62cd7fa68f8b216c0b2340"
   revision 1
```
Comment (DomT4) [inline Library/Formula/httpie.rb]: Please remove this line on stable version updates as well, Thanks!

File: Library/Formula/httpie.rb [hunk 2]
```diff
@@ -1,11 +1,11 @@
 class Httpie < Formula
   desc "User-friendly cURL replacement (command-line HTTP client)"
-  homepage "https://github.com/jakubroztocil/httpie"
-  url "https://github.com/jakubroztocil/httpie/archive/0.9.2.tar.gz"
-  sha256 "1f0b0b6563de25bad0ed190d62f1130573124201382824756d015b1422a234cc"
+  homepage "http://httpie.org/"
```
Comment (DomT4) [inline Library/Formula/httpie.rb]: This seems to just redirect back to `https://github.com/jkbrzt/httpie` at the moment. Is the plan to change that?

Comment (jkbrzt) [inline Library/Formula/httpie.rb]: That is the canonical project homepage URL. Yes, it currently redirects to GitHub, but that might change in the future.

File: Library/Formula/httpie.rb (hunk 1 above)
Comment (jkbrzt) [inline Library/Formula/httpie.rb]: You mean the `revision 1`?

Comment (DomT4) [inline Library/Formula/httpie.rb]: Yup!

File: Library/Formula/httpie.rb (hunk 2 above)
Comment (DomT4) [inline Library/Formula/httpie.rb]: Cool, Thanks for the clarification.

File: Library/Formula/httpie.rb (hunk 1 above)
Comment (jkbrzt) [inline Library/Formula/httpie.rb]: Done!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**